### DISCUSSION TITLE
How can i know the commit_id about official whl release version

### DISCUSSION THREAD

**Discussion by EasonChow001** [microsoft/onnxruntime#7159] [Other Q&A] [answered]
One Questions:
How can i know the commit_id about official whl release version?

**Comment by pranavsharma** [comment, marked as answer]
Every release has a corresponding branch in ORT. For e.g. this is the release branch for 1.7.0 https://github.com/microsoft/onnxruntime/tree/rel-1.7.0. You can get the commit ids from the branch.
